test/data/validWSDLs11/2namespaceSameURL.wsdl

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
What are we asserting for here? That this should correctly get converted? Issue was that body doesn't have data resolved correctly. Not seeing that being asserted here, can we add more extensive tests?